components/atoms/MoreInfo.js

```diff
@@ -0,0 +1,38 @@
+import Link from "next/link";
+import PropTypes from "prop-types";
+import { faArrowAltCircleRight } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+/**
+ *  More Information Component
+ */
+
+export default function MoreInfo(props) {
+  const googleSearch = "https://google.ca/search?q="
+    .concat(props.text)
+    .replace(/\s/g, "+");
+
+  return (
+    <div className="mt-4 mb-4 pb-2 pt-2 px-3 border border-custom-blue-dark rounded text-custom-blue-dark cursor-pointer hover:bg-gray-light-report focus:ring-1 focus:ring-black focus:ring-offset-1 active:bg-gray-active">
+      <Link href={googleSearch}>
```

Comment:
just an FYI -- nothing wrong with using the Next `Link` component but it doesn't help you at all if you are doing external links. If you are doing links between pages on your site, that's when it speeds things up.

Reply:
Good to know!